### DISCUSSION TITLE
Permission-Problem

### DISCUSSION THREAD

**Discussion by Mohammad-Rsi** [Security-Onion-Solutions/securityonion#10196] [General]
I have Windows 10 Enterprise
I Download The Security-Onion Latest version  which we have on GitHub right now 
actually, I wanted to run it up on the virtual box on Windows then I got a message about the permission  which says that you don't have permission to open this file 
even if I can't move it to another directory again I don't have permission.
you also included key file and signature but i don't know how to use them and i couldn't find any insturcture to use them on windows 10
i would very much like instructure if any available

**Comment by ironbract** [comment]
You've probably downloaded the ISO of the installation CDROM, which doesn't run directly.

Create a new Linux VM (Centos 7) in VirtualBox. In the VM configuration, set up the CDROM drive to point to the ISO image you downloaded. (Google for instructions on how to do this if you're uncertain.) When you start the VM it should boot from the ISO and take you through the installation process.

**Reply by Mohammad-Rsi** [reply]
Thanks for response 
yeah I download the iso file
I have one problem which is when I want to give the address to VM while I click on the file to submit the address the message shows to me: you don't have permission to open this file!!!!
